### PR TITLE
DAITDS-240 - DNS View added

### PR DESCRIPTION
**Pull request by ThiagoYU**

### CODE REVIEW

File: rest/model/dns/dns_view.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	Updated_at  int      `json:"updated_at,omitempty"`
+	Read_acls   []string `json:"read_acls"`
+	Update_acls []string `json:"update_acls"`
+	Zones       []string `json:"zones"`
+	Networks    []int    `json:"networks"`
+	Preference  int      `json:"preference,omitempty"`
+}
+
+// NewDNSView takes a view_name and creates a *DNSView
+func NewDNSView(view_name string) *DNSView {
```
Comment (Zach-Johnson) [inline rest/model/dns/dns_view.go]: no underscores in var names

File: rest/dns_view.go [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+		return resp, err
+	}
+
+	return resp, nil
+}
+
+// Get takes a DNS view name and returns DNSView struct.
+//
+// NS1 API docs: https://ns1.com/api#getview-dns-view-details
+func (s *DNSViewService) Get(view_name string) (*dns.DNSView, *http.Response, error) {
```
Comment (Zach-Johnson) [inline rest/dns_view.go]: no underscores in var names

File: rest/dns_view.go [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+		return resp, err
+	}
+
+	return resp, nil
+}
+
+// Delete takes a DNS view name, and removes an existing DNS view
+//
+// NS1 API docs: https://ns1.com/api#deletedelete-a-dns-view
+func (s *DNSViewService) Delete(view_name string) (*http.Response, error) {
```
Comment (Zach-Johnson) [inline rest/dns_view.go]: no underscores in var names

File: rest/dns_view.go [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+// NS1 API docs: https://ns1.com/api#postedit-dns-view-preference
+func (s *DNSViewService) UpdatePreferences(m map[string]int) (map[string]int, *http.Response, error) {
+	path := "config/views/preference"
+
+	req, err := s.client.NewRequest("POST", path, m)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	m_updated := make(map[string]int)
```
Comment (Zach-Johnson) [inline rest/dns_view.go]: no underscores in var names

**Review by Zach-Johnson — CHANGES_REQUESTED** [review]
small cleanup requests

**Comment by eravin-ns1** [comment]
This code was merged in #171

**Pull request closed without merging**